clusterdock/topologies/cdh/actions.py

```diff
@@ -77,13 +100,26 @@ def start(args):
                             **{'volumes_from': [java_image]} if args.java else {})
                        for hostname in args.secondary_nodes]
 
+    if args.kerberos:
```

Comment:
Can we change the order of this block so that it comes after the node groups are defined (so that we don't keep having to have `if args.kerberos:` checks in so many places?